web/template/admin/admin_tabs/stats.gohtml

```diff
@@ -1,137 +1,86 @@
-                    x-show="open"
-                    x-transition.origin.top.left
-                    x-on:click.outside="close($refs.button)"
-                    :id="$id('dropdown-button')"
-                    style="display: none;"
-                    class="absolute right-0 mt-2 w-40 text-white bg-gray-500 dark:bg-gray-900 rounded shadow-md overflow-hidden"
-                >
-                    <div>
-                        <a :href="admin.getStatsDownloadLink('json')" x-on:click="close($refs.button);" class="block w-full px-4 py-2 text-left text-sm dark:hover:bg-gray-600 cursor-pointer" download>
-                            Export as JSON
-                        </a>
-                        <a :href="admin.getStatsDownloadLink('csv')" x-on:click="close($refs.button);"  class="block w-full px-4 py-2 text-left text-sm dark:hover:bg-gray-600 cursor-pointer">
-                            Export as CSV
-                        </a>
-                    </div>
-                </div>
-            </div>
-        </div>
+<h1 class="text-2xl text-1 my-4">Server Statistics</h1>
+<div class="grid grid-cols-1 md:grid-cols-2 gap-4 bg-gray-100 relative rounded-md shadow-md p-6 dark:bg-secondary">
 
-        <div class="col-span-4">
-            <h2 class="text-2 font-medium">Quick stats</h2>
-            <table class="m-2 text-sm">
-                <tbody>
-                <tr>
-                    <td class="text-3 px-1 font-medium">Enrolled Students</td>
-                    <td class="text-2 px-1 font-semibold" id="numStudents">
-                        <i class="fas fa-spinner animate-spin text-3"></i>
-                    </td>
-                </tr>
-                <tr>
-                    <td class="text-3 p-1 font-medium">Lectures</td>
-                    <td class="text-2 px-1 font-semibold">{{.TUMLiveContext.Course.NumStreams}}</td>
-                </tr>
-                <tr>
-                    <td class="text-3 px-1 font-medium">Vod Views</td>
-                    <td class="text-2 font-semibold px-1" id="vodViews">
-                        <i class="fas fa-spinner animate-spin text-3"></i>
-                    </td>
-                </tr>
-                <tr>
-                    <td class="text-3 px-1 font-medium">Live Views</td>
-                    <td class="text-2 font-semibold px-1" id="liveViews">
-                        <i class="fas fa-spinner animate-spin text-3"></i>
-                    </td>
-                </tr>
-                </tbody>
-            </table>
-        </div>
-        <div class="col-span-4 sm:col-span-2">
-            <h2 class="text-2 font-medium">Student Live activity per week</h2>
-            <div class="w-full m-auto" style="min-height: 200px">
-                <canvas id="courseGeneralStatsLive" width="400" height="100" aria-label="Viewer Live stats"
-                        role="img"></canvas>
-            </div>
+    <h2 class="text-2 font-bold col-span-full">Quick stats</h2>
```

Comment:
For consistency, make sure that each title does not have the `font-medium` class. For example, the "Worker Overview" still has this class in its classlist.